base/sparse/spqr.jl

```diff
@@ -21,133 +21,293 @@ const ORDERING_BESTAMD = Int32(9) # try COLAMD and AMD; pick best#
 # tried.  If there is a high fill-in with AMD then try METIS(A'A) and take
```

Comment:
I don't really remember why I included this comment. I don't think it is useful so I'll just delete it.